### ISSUE TITLE
core::fmt::String is not implemented for the type CreationError

### ISSUE THREAD

**Issue by obsgolem** [tomaka/glutin#225]
I am trying to compile the following:
```
extern crate glutin;

fn main() {
	let window = glutin::WindowBuilder::new().with_dimensions(640, 640).build().unwrap();
	unsafe {window.make_current()};


	while !window.is_closed() {
		window.wait_events();

		window.swap_buffers();
	}
}
```
When I do I get this error:
```
   Compiling glutin v0.0.4-pre (https://github.com/tomaka/glutin#bd3d7513)
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:76:1: 83:2 error: the trait `core::fmt::String` is not implemented for the type `CreationError` [E0277]
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:76 impl std::error::Error for CreationError {
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:77     fn description(&self) -> &str {
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:78         match self {
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:79             &CreationError::OsError(ref text) => text.as_slice(),
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:80             &CreationError::NotSupported => "Some of the requested attributes are not supported",
C:\Users\josia_000\.cargo\git\checkouts\glutin-666c1766c69e6584\master\src\lib.rs:81         }
                                                                                     ...
error: aborting due to previous error
Could not compile `glutin`.
```
This is my version string:
```
rustc 1.0.0-nightly (4874ca36f 2015-01-23 00:18:57 +0000)
binary: rustc
commit-hash: 4874ca36f61d1975398c2ddb9e3d43443a8c373b
commit-date: 2015-01-23 00:18:57 +0000
host: x86_64-pc-windows-gnu
release: 1.0.0-nightly
```

**Comment by tomaka** [comment]
Probably fixed now thanks to #224

**Comment by obsgolem** [comment]
It seems to be for me.
